src/Purebred.hs

```diff
@@ -99,7 +100,10 @@ launch cfg = do
     void $ defaultMain (theApp s) s
 
 processConfig :: UserConfiguration -> IO InternalConfiguration
-processConfig = (confNotmuch . nmDatabase) id
+processConfig cfg = do
+    fp <- view (confNotmuch . nmDatabase) cfg
+    ed <- view confEditor cfg
+    pure $ cfg & set (confNotmuch . nmDatabase) (fp) & set confEditor (ed)
```

Comment:
would be good to break this across multiple lines & remove extraneous parens, e.g.
```haskell
pure $ cfg
  & set (confNotmuch . nmDatabase) fp
  & set confEditor ed
```

Reply:
Ah dang! Yes of course. Will do.